packages/frontend2/src/app/(legacy)/layout.tsx

```diff
@@ -4,7 +4,7 @@ import { NavLayout } from '../_components/nav/nav-layout'
 export default function Layout({ children }: { children: React.ReactNode }) {
   return (
     <NavLayout logoLink="/" legacyNav>
-      <main className="min-h-screen pb-20">{children}</main>
+      <div className="min-h-screen pb-20">{children}</div>
```

Comment:
Is there a case where we will not want this to be `main`?

Reply:
yup, basically children can return also a `header` and it should be placed outside of `main` tag